src/main/java/org/jenkinsci/plugins/xunit/threshold/XUnitThreshold.java

```diff
@@ -94,27 +94,28 @@ public Result getResultThresholdNumber(XUnitLog log,
                                            int testCount,
                                            int newTestCount) {
 
+        String errorMessage = "The total number of tests for this category exceeds the specified '%s' threshold value.";
         if (isValid(getFailureThreshold())
                 && (convertToInteger(getFailureThreshold()) < testCount)) {
-            log.infoConsoleLogger("The total number of tests for this category exceeds the specified 'failure' threshold value.");
+            log.infoConsoleLogger(String.format(errorMessage, "failure"));
             return Result.FAILURE;
         }
 
         if (isValid(getFailureNewThreshold())
                 && (convertToInteger(getFailureNewThreshold()) < newTestCount)) {
-            log.infoConsoleLogger("The new number of tests for this category exceeds the specified 'new failure' threshold value.");
```

Comment:
The messaging isn't all the clear to begin with so I wouldn't say worse. In fact, the change is not even wrong. The total number of tests for a category could be the total number of new failure tests in a category. I'll was changing back to the original because @v1v makes a good point about people depending on it but a better message would say "The number of tests for this category exceeds..."